tile.py

```diff
@@ -53,7 +53,22 @@ def kill_robot(self, robot):
         """
         return robot
 
-    def move_robot(self, robot, state):
+    def check_belts(self, express_belts):
+        """
+        Check that current tile is conveyor belt of desired type.
+        express_belts: a boolean, True for express belts, False for all belts.
+        Return a boolean.
+        True - Tile is conveyor belt of desired type.
+        False - Tile isn't conveyor belt or it's a wrong type of belt.
+        """
+        return False
+
+    def rotate_robot_on_belt(self, robot, direction):
+        """
+        Rotate robot on rotating conveyor belts. If robot will be rotated,
```

Comment:
```suggestion
        Rotate robot on rotating conveyor belts. Robot's rotation
```

Reply:
No podle toho směru se vyhodnotí, jestli se robot vůbec otočí a pokud ano, tak jak.

Reply:
Tak třeba?
`If robot's rotated, will be decided by the direction he entered a tile.`
To je jen grammarnazi - conditionals nedovolují budoucí čas v "if" větě.